### PR TITLE
Created split.js and implemented my own String.prototype.split() function 

### PR DESCRIPTION
**Pull request by terabo090**

### CODE REVIEW

**Comment by terabo090** [comment]
@TheSTL i have made few changes,
plz review!!

File: implementations/split.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      nextVal += similarSplit;
+    }
+    i = i + split_length_passed;
+  } else {    // When the current character is different from `splitVal` 's first character
+    nextVal += string[i];
+    i++;
+  }
+}
+ outputArr.push(nextVal);
+ outputArr=outputArr.filter((data,i)=> return i < limit);
```
Comment (TheSTL) [inline implementations/split.js]: In SINGLE EXPRESSION RETURN you don't have to write `return` keyword.
`outputArr=outputArr.filter((data,i)=> i < limit);`

File: implementations/split.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+//str = "val1|val2|val3"
+//str.split('|',2);
+// output
+// ["val1", "val2"]
+
+
+String.prototype.split = function(splitVal,limit) {
+  const outputArr = [];
```
Comment (TheSTL) [inline implementations/split.js]: It should be let because you are reassign the variable

**Comment by TheSTL** [comment]
I'm closing this PR.
Merged PR is - [#35]

**Pull request closed without merging**